dynamo/plot/utils.py

```diff
@@ -346,19 +356,21 @@ def _matplotlib_points(
                 levels=100,
             )
             x, y = points[:, :2].T
-            ax.scatter(
-                x,
-                y,
+            _scatter_projection(
+                ax,
+                points,
+                projection,
                 c=colors,
                 plotnonfinite=True,
                 zorder=21,
                 **kwargs,
             )
         else:
             main_debug("drawing without frontiers and contour")
-            ax.scatter(
-                points[:, 0],
-                points[:, 1],
+            _scatter_projection(
+                ax,
+                points,
+                projection,
```

Comment:
should we also add rasterized=rasterized, here?

Reply:
The old code does not have rasterized=rasterized here. I am not sure if I add some argument to ~2000 lines of code function "scatters" can cause any problem.

Reply:
see above